src/api/pipeline/schema.capnp

```diff
@@ -0,0 +1,111 @@
+@0xbf46af72b205a04b;
+
+struct JobStatus {
+  id          @0 :Text;
+  description @1 :Text;
+  canCancel   @2 :Bool;
+  canRebuild  @3 :Bool;
+}
```

Comment:
It would be really nice to have a documentation generator from this `schema` to HTML / Markdown. 
That might be an interesting hacking day project to build.